docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,281 @@
+
+The `TwoOrgsOrdererGenesis:` profile creates a consortium named `SampleConsortium` that contains the two peer organizations in the `configtx.yaml` file, Org1 and Org2. The `Orderer` section of the profile uses the single node raft ordering service defined in the **Orderer:** section of the file. The OrdererOrg from the **Organizations:** section is made the only administrator of the ordering service. Because our only ordering node is running Fabric 2.x, we can set the orderer system channel capability to `V2_0`. The system channel uses default policies from the **Channel** section and enables `V2_0` as the channel capability level.
+
+The `TwoOrgsChannel:` profile is used to create application channels.
+```yaml
+TwoOrgsChannel:
+    Consortium: SampleConsortium
+    <<: *ChannelDefaults
+    Application:
+        <<: *ApplicationDefaults
+        Organizations:
+            - *Org1
+            - *Org2
+        Capabilities:
+            <<: *ApplicationCapabilities
+```
+
+The system channel is used by the ordering service as a template to create application channels. The nodes of the ordering service that are defined in the system channel become the default consenter set of new channels. The channel MSPs for each organization is transferred to the new channel from the system channel. After the channel is created, ordering nodes can be added or removed from the the channel by updating the channel configuration. The same process can also be used to [add a channel member](../channel_update_tutorial.html) that does not belong to the consortium.
+
+The `TwoOrgsChannel` provides name of the `SampleConsortium` that is used to create the channel. As a result, the ordering service defined in the `TwoOrgsOrdererGenesis` profile becomes channel consenter set. The `Application` section creates the part of the channel that governs the channel's peer organizations. Both organizations from the consortium, Org1 and Org2, are included as channel members. The channel uses `V2_0` as the application capabilities, and uses the default policies to govern how peer organizations will interact with the channel. The application channel also uses the default policies from the **Channel** section and enables `V2_0` as the channel capability level.
+
+## Channel policies
+
+Because channels are used as a private method of communication between organizations, any changes that are made to the channel need to be agreed by other channel members. A channel would not be useful if an organization could join the channel and read the data on the ledger without getting the approval of other organizations. Because part of the channel ledger is stored on the ordering nodes that create new blocks, channel members also need to select the ordering nodes that participate in the channel. Any changes to the **structure** of the channel need to be approved by the set of organizations that is specified by channel policies.
```

Comment:
I don't understand what this second sentence is getting at. Joining/reading the channel is a fundamentally different activity from changing the channel config.

Reply:
simplified this.